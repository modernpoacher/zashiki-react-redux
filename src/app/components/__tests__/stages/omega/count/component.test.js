import 'jsdom-global/register'

import React from 'react'
import Enzyme, { mount } from 'enzyme'
import toJson from 'enzyme-to-json'
import Adapter from 'enzyme-adapter-react-16'

import Count from '@modernpoacher/zashiki-react-redux/app/components/stages/omega/count/component'

Enzyme.configure({ adapter: new Adapter() })

describe('@modernpoacher/zashiki-react-redux/app/components/stages/omega/count/component', () => {
  it('renders', () => {
    const component = (
      <Count index={1} count={2} />
    )

    expect(toJson(mount(component)))
      .toMatchSnapshot()
  })
})
