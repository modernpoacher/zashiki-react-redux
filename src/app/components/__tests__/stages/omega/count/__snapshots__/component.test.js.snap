// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@modernpoacher/zashiki-react-redux/app/components/stages/omega/count/component renders 1`] = `ReactWrapper {}`;
